docs/examples.rst

```diff
@@ -677,6 +677,37 @@ variables or context dimensions:
         ref: $@distro
 
 
+.. _anchors-aliases:
+
+Using anchors and aliases in Plans
+------------------------------------------------------------------
+
+When you need to specify the same variable multiple times,
+YAML has a feature called `anchors and aliases`__. You can define an
+anchor before an item to save it for future usage with an `alias`__.
+
+Example of an anchor:
+
+.. code-block:: yaml
+
+    discover:
+        how: fmf
+        test: &stable
+            - first
+            - second
+
+Which you can then use later in the same file as alias:
+
+.. code-block:: yaml
```

Comment:
What about presenting the example as a single block and using `yaml` comments for the two descriptions? At the first sight I got the impression that this can be used in two different files.